### PR TITLE
Use inventory as component from insigths frontend components

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

**Comment by karelhala** [comment]
ping @jharting

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
     componentDidMount () {
         insights.chrome.init();
-        insights.chrome.identifyApp('inventory');
+        insights.chrome.identifyApp(INVENTORY_ROOT);
+        this.appNav = insights.chrome.on('APP_NAVIGATION', event => this.props.history.push(`/${event.navId}`));
+    }
+
+    componentWillUnmount() {
+        this.appNav();
+        this.buildNav();
```
Comment (jharting) [inline src/App.js]: where does `buildNav` come from?

File: config/spandx.config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* global exports */
 
 exports.routes = {
-    '/entities': { host: 'http://insights-inventory-platform-ci.1b13.insights.openshiftapps.com' }
+  '/hosts': { host: 'http://host-inventory-insights-host-inventory.5a9f.insights-dev.openshiftapps.com/api' }
```
Comment (jharting) [inline config/spandx.config.js]: this should be reflected in config.js

File: src/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en-US">
     <head>
         <meta charset="UTF-8">
-        <title>Red Hat Insights | Inventory</title>
+        <title>Red Hat Insights | Sample App</title>
```
Comment (jharting) [inline src/index.html]: :-/

File: src/routes/Inventory.js [hunk 4]
```diff
@@ -1,43 +1,92 @@
...
+    }
+
+    async loadInventory() {
+        const {
+            inventoryConnector,
+            INVENTORY_ACTION_TYPES,
+            mergeWithEntities,
+            mergeWithDetail
+        } = await asyncInventoryLoader();
+        // eslint-disable-next-line no-console
```
Comment (jharting) [inline src/routes/Inventory.js]: no-console?

File: config/spandx.config.js (hunk 2 above)
Comment (karelhala) [reply config/spandx.config.js]: Good point!

File: src/App.js (hunk 1 above)
Comment (karelhala) [reply src/App.js]: Leftover from recent changes, good point!

**Pull request merged**